src/content/getusermedia/resolution/js/test.js

```diff
@@ -67,3 +67,7 @@ test('HD capture', function(t) {
 test('FULL HD capture', function(t) {
   resolutionTest(t, 'full-hd', 1920);
 });
+
+test('Four K capture', function(t) {
```

Comment:
Here you can have "4K" since it's in a string.

Reply:
But I think either 4K is not enabled on Chrome or the fake camera capture device does not support 4K hence the test times out. I think you can comment out this test and add a comment saying that the fake camera capture device does not support 4K.

You could try this out locally and see if 4K works with a 4K camera at all by modifying the values for the Full HD button on https://webrtc.github.io/samples/src/content/getusermedia/resolution/ in the browser before you click the full HD button. If that works, then try the same using the fake camera ( start Chrome with the command line flag: --use-fake-device-for-media-stream).